src/components/PeopleFilters.tsx

```diff
@@ -16,77 +82,72 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={query}
+            onChange={handleChangeQuery}
           />
 
           <span className="icon is-left">
-            <i className="fas fa-search" aria-hidden="true" />
+            <i
+              className="fas fa-search"
+              aria-hidden="true"
+              onClick={() => handleChangeQuery()}
```

Comment:
```suggestion
              onClick={handleChangeQuery}
```

Reply:
As for me, it's better to leave this as it is, because in another way I need to add an additional event type and check for handleChangeQuery func